### PR TITLE
Add Non-recommended schemes to PKM overview

### PR DESCRIPTION
**Pull request by Bosch-0**
[Preview](https://deploy-preview-579--sad-borg-390916.netlify.app/guide/private-key-management/overview/#non-recommended-schemes)

Closes https://github.com/BitcoinDesign/Guide/issues/398

### CODE REVIEW

File: guide/private-key-management/overview.md [hunk 1]
```diff
@@ -127,6 +127,14 @@ The use cases for shared schemes include spouses managing a joint account, group
 
 - [Multi-key](/guide/private-key-management/multi-key/) - several keys and several people control the wallet
 
+### Non-recommended schemes
+
+There are also several non-recommended private key schemes not detailed in this guide that are not widely used due to their poor security, lack of interoperability and unnecessary complexity. Below are schemes not recommended to be used in your Bitcoin applications and why.
+
+- Key sharding (Shamir's secret sharing): Sometimes referred to as an alternative to multi-key setups though the security risks are much greater. It's major drawbacks are a single point of failure, no key invalidation, no widely used implementation (though [SLIP-0039](https://github.com/satoshilabs/slips/blob/master/slip-0039.md) does exist), and poor auditability. Read more about the issues with key sharding [here](https://docs.keys.casa/wealth-security-protocol/rejected-key-schemes/key-sharding-shamirs-secret-sharing).
```
Comment (GBKS) [inline guide/private-key-management/overview.md:134]:
```suggestion
- Key sharding (Shamir's secret sharing): Sometimes referred to as an alternative to multi-key setups, but the security risks are much greater. It's major drawbacks are a single point of failure, no key invalidation, no widely used implementation (though [SLIP-0039](https://github.com/satoshilabs/slips/blob/master/slip-0039.md) does exist), and poor auditability. Read more about the issues with key sharding [here](https://docs.keys.casa/wealth-security-protocol/rejected-key-schemes/key-sharding-shamirs-secret-sharing).
```

**Comment by GBKS** [comment]
ACK. Just one minor tweak. Wonder if this could be extracted to a sub-page of the glossary, so this page could stay simple.

**Comment by Bosch-0** [comment]
@GBKS it's such a small addition I don't need it needs it's own page

File: guide/private-key-management/overview.md [hunk 2]
```diff
@@ -127,6 +127,14 @@ The use cases for shared schemes include spouses managing a joint account, group
 
 - [Multi-key](/guide/private-key-management/multi-key/) - several keys and several people control the wallet
 
+### Non-recommended schemes
+
+There are also several non-recommended private key schemes not detailed in this guide that are not widely used due to their poor security, lack of interoperability and unnecessary complexity. Below are schemes not recommended to be used in your Bitcoin applications and why.
+
+- Key sharding (Shamir's secret sharing): Sometimes referred to as an alternative to multi-key setups, but the security risks are much greater. It's major drawbacks are a single point of failure, no key invalidation, no widely used implementation (though [SLIP-0039](https://github.com/satoshilabs/slips/blob/master/slip-0039.md) does exist), and poor auditability. Read more about the issues with key sharding [here](https://docs.keys.casa/wealth-security-protocol/rejected-key-schemes/key-sharding-shamirs-secret-sharing).
```
Comment (pavlenex) [inline guide/private-key-management/overview.md:134]:
```suggestion
- Key sharding (Shamir's secret sharing): Sometimes referred to as an alternative to multi-key setups, but the security risks are much greater. Its major drawbacks are a single point of failure, no key invalidation, no widely used implementation (though [SLIP-0039](https://github.com/satoshilabs/slips/blob/master/slip-0039.md) does exist), and poor auditability. Read more about the issues with key sharding [here](https://docs.keys.casa/wealth-security-protocol/rejected-key-schemes/key-sharding-shamirs-secret-sharing).
```

File: guide/private-key-management/overview.md [hunk 3]
```diff
@@ -127,6 +127,14 @@ The use cases for shared schemes include spouses managing a joint account, group
...
+- DAT files: A legacy form of backing up bitcoin private keys that use the generic .dat file type. These are are still used today by [Bitcoin Core](https://bitcoincore.org/) due to [security concerns](https://bitcoin.stackexchange.com/questions/92716/why-keys-generated-in-bitcoin-core-mismatch-that-from-online-generators-despite) with non-hardened key derivation. This form of private key scheme is not widely adopted due to it's lack of interoperability among wallets, not being human readable like a recovery phrase, and being potentially insecure if the user does not encrypt the keys contained within the file.
```
Comment (pavlenex) [inline guide/private-key-management/overview.md:135]:
```suggestion
- DAT files: A legacy form of backing up Bitcoin private keys that use the generic .dat file type. These are still used today by [Bitcoin Core](https://bitcoincore.org/) due to [security concerns](https://bitcoin.stackexchange.com/questions/92716/why-keys-generated-in-bitcoin-core-mismatch-that-from-online-generators-despite) with non-hardened key derivation. This form of private key scheme is not widely adopted due to its lack of interoperability among wallets, not being human readable like a recovery phrase, and being potentially insecure if the user does not encrypt the keys contained within the file. 
```

**Review by GBKS — APPROVED** [review]
ACK. Good addition, so it doesn't look like we missed certain schemes.

**Pull request merged**